Makefile

```diff
@@ -140,6 +140,12 @@ BUILD_TARGETS = bin/git-lfs-darwin-amd64 bin/git-lfs-darwin-386 \
 	bin/git-lfs-freebsd-amd64 bin/git-lfs-freebsd-386 \
 	bin/git-lfs-windows-amd64.exe bin/git-lfs-windows-386.exe
 
+.PHONY : mangen
+mangen : $(GOPATH)/src/github.com/git-lfs/git-lfs/commands/mancontent_gen.go
+
+$(GOPATH)/src/github.com/git-lfs/git-lfs/commands/mancontent_gen.go :
```

Comment:
This rule look great! I had two quick notes.

Firstly, could you change this line from the fully-qualified GOPATH-style to just `commands/mancontent_gen.go`? Make will interpret this correctly, and it matches the style of other recipes in this file.

Secondly, could you please add documentation about what this recipe does? You can look to other recipes in this file for inspiration about what is standard practice here. Thanks.